src/features/device-logs/index.ts

```diff
@@ -22,6 +22,18 @@ const deviceLogsRateLimiter = createRateLimitMiddleware(
 	},
 );
 
+// Rate limit for device log creation, a maximum of 15 batches every 10 second window
+const streamableDeviceLogsRateLimiter = createRateLimitMiddleware(
+	createRateLimiter('get-device-logs', {
+		points: 10, // allow 10 device log streams / requests batches
+		blockDuration: 60, // seconds
+		duration: 60, // reset counter after 10 seconds (from the first batch of the window)
```

Comment:
The comment of "10 seconds" doesn't match the value of "60". Forgot to update copy pasted comment?

Reply:
Thx!